--- ./gtkDPS/Makefile.am.org	Mon Nov 26 14:25:17 2001
+++ ./gtkDPS/Makefile.am	Mon Nov 26 14:30:52 2001
@@ -48,13 +48,13 @@
 
 libgtkDPS_la_LDFLAGS = -version-info 1:0:0
 
-AM_CFLAGS+= $(DPS_CFLAGS)         \
+AM_CFLAGS = $(DPS_CFLAGS)         \
 	-DLOCALEDIR=\""$(gtkdpslocaledir)"\"	\
 	$(GTK_CFLAGS)
 
 INCLUDES = -I$(top_srcdir)
 
-LDADD+= libgtkDPS.la $(DPS_LIBS) $(GTK_LIBS)
+LDADD = libgtkDPS.la $(DPS_LIBS) $(GTK_LIBS)
 
 noinst_PROGRAMS=intersect sample sample2 dpsmark fontsel paintsel linesel
 
